src/library_pthread.js

```diff
@@ -1240,6 +1236,24 @@ var LibraryPThread = {
 #endif
   },
 
+  $exitProcess__deps: ['$handleException', 'exit'],
```

Comment:
Should the `$handleException` symbol here removed when linking with `-sMINIMAL_RUNTIME`? That should correspond to this logic.
https://github.com/emscripten-core/emscripten/blob/0dde48f10e715b1d4013fe5128acb9b897ff0ff4/src/library_pthread.js#L13-L15

And would probably fix the `browser.test_minimal_runtime_hello_thread*` failures.
```
test_minimal_runtime_hello_thread (test_browser.browser) ... error: undefined symbol: $handleException (referenced by $exitProcess__deps: ['$handleException','exit','$PThread'], referenced by top-level compiled C/C++ code)
```
(untested)

Reply:
Indeed! Thanks.